gui/mozregui/mainwindow.py

```diff
@@ -75,6 +76,13 @@ def __init__(self):
         self.ui.log_view.setVisible(True)
         self.ui.logDockWidget.setVisible(True)
 
+        # Use better toolbar icons in dark mode
+        dark_mode_enabled = QGuiApplication.palette().color(QPalette.Window).lightness() < 128
+        if dark_mode_enabled:
+            self.ui.actionRun_a_single_build.setIcon(QIcon(":/s_white.png"))
+            self.ui.actionStart_a_new_bisection.setIcon(QIcon(":/cutting_white.png"))
+            self.ui.actionStop_the_bisection.setIcon(QIcon(":/stop_white.png"))
```

Comment:
To follow existing conventions, it would be better to set these icons in `gui/mozregui/mainwindow.py`, replacing the existing `icon`, `icon1`, and `icon2` values for the filename, or possibly creating "dark" versions of those variables and using those instead.

Reply:
> it would be better to set these icons in `gui/mozregui/mainwindow.py`

For clarification, do you mean `gui/mozregui/ui/mainwindow.py`? That's the only place where I was able to find `icon1` etc but that seems to be auto generated. This file is already `gui/mozregui/mainwindow.py`

Reply:
Realized that I never replied to this.

> For clarification, do you mean `gui/mozregui/ui/mainwindow.py`? That's the only place where I was able to find `icon1` etc but that seems to be auto generated and I'm not sure I can accomplish this in the corresponding `.ui` file. This file is already `gui/mozregui/mainwindow.py`

Yes you are correct -- what I meant to say is "in the corresponding `.ui` file" of `gui/mozregui/ui/mainwindow.py`. That is probably nontrivial as you mentioned (if at all possible) since there isn't a context we can assign the different icons to in that file (even though multiple images can be assigned to the same icon).

A possible alternate approach without adding new icons that aren't in the `.ui` file could be to change the colour of the bitmap, e.g., by means of using a mask, or inverting the colours. But I think what you have right now is good as well.